Microservices.CommandService/appsettings.Production.json

```diff
@@ -0,0 +1,8 @@
+{
+  "ConnectionStrings": {
+    "MSSqlDatabaseProduction": "Server=mssql-clusterip-service,1433;Database=CommandService;Initial Catalog=commanddb;User ID=sa;Password=pass55word!"
```

Comment:
Connection string should not contain sensitive data.

Reply:
Ok, where I should store this data?

Reply:
Use environment variables to insert username + password.